subtitles_test.go

```diff
@@ -340,3 +340,26 @@ func TestHTMLEntity(t *testing.T) {
 		assert.Equal(t, string(c), w.String())
 	}
 }
+
+func TestNewScanner(t *testing.T) {
+	exts := []string{"vtt", "srt", "ssa"}
+	for _, ext := range exts {
+		s, err := astisub.OpenFile("./testdata/example-in-scan-line." + ext)
```

Comment:
Could you rename the testdata files to `example-in-carriage-return.(srt|ssa|vtt)` instead?